### PR TITLE
Changing state for transfer

### PR DESCRIPTION
**Pull request by truthfool**
## Description

- changing state from IN_PROGRESS to COMPLETED

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj

File: src/main/java/org/mifos/connector/mpesa/camel/routes/PaybillRoute.java [hunk 1]
```diff
@@ -199,6 +200,7 @@ public void configure() {
                     variables.put("phoneNumber", paybillConfirmationRequestDTO.getMsisdn());
                     variables.put("mpesaTransactionId", mpesaTransactionId);
                     variables.put(TRANSACTION_ID, transactionId);
+                    variables.put(TRANSFER_CREATE_FAILED, false);
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/mpesa/camel/routes/PaybillRoute.java:203]: how does it move to completed
?

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Comment by fynmanoj** [comment]
please add more design/resolution points

File: src/main/java/org/mifos/connector/mpesa/camel/routes/PaybillRoute.java (hunk 1 above)
Comment (truthfool) [reply src/main/java/org/mifos/connector/mpesa/camel/routes/PaybillRoute.java:203]: This is after firing the confirmation request.
Validation : IN_PROGRESS
Confirmation : COMPLETED

**Comment by truthfool** [comment]
The transfer and transaction requests have different sets of variables to be published which moves them from different states based on importer rdbms code. As paybill is a transfer request and buygoods is txn request this variable is for transfer request.

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**